src/i18n/getLocalizedPath.js

```diff
@@ -0,0 +1,15 @@
+const languages = require("./languages");
+
+const defaultLanguage = languages.find(lang => lang.default);
```

Comment:
What happen If user forget to set any language as a `default`? 🙂

Reply:
there would be no `index.html` etc. just `/en/index.html`, `/cs/index.html` etc.

Reply:
and build will crash because of defaultLanguage.routes[keyPath] 😄